hcl2template/repl/format.go

```diff
@@ -0,0 +1,106 @@
+package repl
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"sort"
+	"strconv"
+	"strings"
+)
+
+// FormatResult formats the given result value for human-readable output.
+//
+// The value must currently be a string, list, map, and any nested values
+// with those same types.
+func FormatResult(value interface{}) string {
```

Comment:
interesting! this can be useful for other scopes in the future :D

Reply:
yes ! :D